jujugui/static/gui/src/app/assets/css/machine-view/_machine-token.scss

```diff
@@ -1,5 +1,16 @@
 .machine-token {
     .token {
+        &.active {
+            margin-right: -21px;
+            padding-right: 31px;
+            border-style: dashed;
+            border-right: 1px solid $canvas-background;
+
```

Comment:
OK, so this actually caused one of my QA issues. I went back to the mocks, scratched my head, and said… "Oh, it's really supposed to be like that. Hmm. That's interesting." I think we should raise it with UX; it looks (to me) like a CSS bug, not intentional design.